hooks/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+echo "*******************************"
+echo   "Running git pre-commit hook"
+echo "*******************************"
+
+./gradlew lintDevDebug
+lintDevDebugStatus=$?
+echo "lintDevDebugStatus Result :"
+if [ "$lintDevDebugStatus" = 0 ]; then
+    echo "No issues were found in lintDevDebug"
+else
+    echo 1>&2 "Need to fix issues before commit."
+    exit 1
+fi
```

Comment:
`-e` option means this script(shell) will abort right after any non-zero code is returned so `$?` without if-statement won't work. 

```suggestion
if ./gradlew lintDevDebug then
    echo "No issues were found in lintDevDebug"
else
    echo "Android Lint exited with $?. Need to fix issues before commit."
    exit 1
fi
```